package.json

```diff
@@ -7,10 +7,9 @@
         "dev": "yarn && nuxi dev",
         "dev:localserver": "NUXT_USE_LOCAL_API=true; yarn && nuxi dev",
         "generate": "graphql-codegen --config ./typesgeneratorconfig.ts",
-        "lint": "ESLINT_USE_FLAT_CONFIG=true eslint . -c eslint.config.js && stylelint **/*.{vue,css,scss} --ignore-path .gitignore",
+        "lint": "ESLINT_USE_FLAT_CONFIG=true eslint . -c eslint.config.js",
```

Comment:
It looks like this was running on `*.vue` files as well. I don't know much about stylelint, so I'll leave this up to your judgment.

Reply:
yup! it was running against "all our css" in our vue files which is 0 because of tailwind 😆